modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -132,15 +132,15 @@ During the _Process_ phase, Mule starts pulling records from the stepping queue
 
 image::batch-diagram.jpg[batch+diagram]
 
-Note that a batch job instance does not wait for all its queued records to finish processing in one batch step before pushing any of them to the next batch step. However, if you configure an aggregator, the batch job step changes its behavior while processing records, depending on how the aggregator is configured:
+Note that by default, a batch job instance does not wait for all its queued records to finish processing in one batch step before pushing any of them to the next batch step. However,  configuring the batch job or the aggregator can change the behavior of the batch job step:
 
 * Behavior with aggregator configured with fixed size
 +
 In this scenario, the batch step sends the processed records to an aggregator, which starts processing the records and buffering them until the configured aggregator's size is reached. After that, the aggregator sends the aggregated records to the stepping queue.
 +
 image:mruntime-batch-job-process-aggregator-fixed.png[Batch job process with Aggregator with fixed size]
 +
-<1> The batch job builds record blocks of the configured block size and sends them to their corresponding batch step for processing.
+<1> The batch job builds blocks of records limited by the configured block size and sends them to their corresponding batch step for processing. The block size is an upper limit. If fewer records are available than the configured size, the batch job also sends the records to the batch step. 
```

Comment:
```suggestion
<1> The batch job builds blocks of records limited by the configured block size and sends them to their corresponding batch step for processing. If fewer records are available than the configured size, the batch job will also send them to the corresponding batch step instead of waiting for more records.
```